react/SlideToggle/SlideToggle.test.js

```diff
@@ -39,12 +39,12 @@ describe('Slide toggle:', () => {
     const slideToggle = renderIntoDocument(
       <SlideToggle
         id="testToggle"
-        text="Test toggle"
+        label="Test toggle"
         onToggle={onToggle}
       />
     );
     const input = findRenderedDOMComponentWithTag(slideToggle, 'input');
     Simulate.change(input);
-    expect(onToggle).toHaveBeenCalledWith(false);
+    expect(onToggle).toHaveBeenCalled();
```

Comment:
Ah, I meant to come back and tidy this up.  I changed the functionality to pass through the entire event object and was struggling to find the doco on how to assert that any object was being passed into the function in Jest.  Have just updated the test.